build.gradle.kts

```diff
@@ -0,0 +1,95 @@
+plugins {
+	id("fabric-loom") version "0.7-SNAPSHOT"
+	`maven-publish`
+}
+
+val sourceCompatibility = JavaVersion.VERSION_1_8
+val targetCompatibility = JavaVersion.VERSION_1_8
+
+val archives_base_name: String by project
+val mod_version: String by project
+val maven_group: String by project
```

Comment:
Why these? Are they no longer automatically polled from `gradle.properties`? Preferable if you poll from the properties in the `{$}` evaluation block.

Reply:
Because that's not really possible in Kotlin hence the type safe part you need to explicitly declare your gradle.properties variables